3rd-party-licenses.md

```diff
@@ -0,0 +1,147 @@
+### Symfony Framework and Symfony Components
+License: MIT  
+https://symfony.com/
+
+### Elasticsearch
+License: Apache License 2.0  
+https://github.com/elastic/elasticsearch-php/blob/master/LICENSE
+
+### Grunt: The JavaScript Task Runner
+License: MIT  
+https://github.com/gruntjs/grunt
+
+### Phing
+License: LGPL-3.0-only  
+https://github.com/phingofficial/phing/blob/master/LICENSE
+
+### Nette Foundation tools 
+License: BSD-2-Clause or GPL-2.0 or GPL-3.0  
+https://nette.org/
+
+### PHP_CodeSniffer
+License: BSD-3-Clause  
+https://github.com/squizlabs/PHP_CodeSniffer
+
+### PHP Parallel Lint
+License: BSD-2-Clause  
+https://github.com/JakubOnderka/PHP-Parallel-Lint/blob/master/LICENSE
+
+### ESLint
+License: MIT  
+https://github.com/eslint/eslint/blob/master/LICENSE
+
+### jQuery
+License: MIT or GPL Version 2  
+http://jquery.org/license
+
+### selectize.js
+License: Apache License 2.0  
+https://github.com/selectize/selectize.js/blob/master/LICENSE
+
+### slick.js
+License: MIT  
+https://github.com/kenwheeler/slick/blob/master/LICENSE
+
+### nodejs-npm
+License: Artistic License 2.0  
+https://www.npmjs.com/policies/npm-license
+
+### gnu-libiconv
+License: GPL v3  
+https://directory.fsf.org/wiki/Libiconv
+
+### eslint-config-standard
+License: MIT  
+https://github.com/standard/eslint-config-standard/blob/master/LICENSE
+
+### Autoprefixer
+License: MIT  
+https://github.com/postcss/autoprefixer/blob/master/LICENSE
+
+### ES6-Promise (subset of rsvp.js)
+License: MIT  
+https://github.com/stefanpenner/es6-promise/blob/master/LICENSE
+
+### time-grunt
+License: MIT  
+https://github.com/sindresorhus/time-grunt/blob/master/license
+
+### jit-grunt 0.10.0
+License: MIT  
+https://github.com/shootaroo/jit-grunt
+
+### SVG to webfont converter for Grunt
+License: MIT  
+https://github.com/sapegin/grunt-webfont/blob/master/License.md
+
+### grunt-spritesmith
+License: MIT  
+https://github.com/twolfson/grunt-spritesmith/blob/master/LICENSE-MIT
+
+### grunt-postcss
+License: MIT  
+https://github.com/nDmitry/grunt-postcss/blob/master/LICENSE
+
+### grunt-legacssy
+License: MIT  
+https://github.com/robinpokorny/grunt-legacssy/blob/master/LICENSE
+
+### eslint-plugin-standard
+License: MIT  
+https://github.com/standard/eslint-plugin-standard/blob/master/LICENSE
+
+### eslint-plugin-promise
+License: ISC  
+https://github.com/xjamundx/eslint-plugin-promise#license
+
+### eslint-plugin-node
+License: MIT  
+https://github.com/mysticatea/eslint-plugin-node/blob/master/LICENSE
+
+### eslint-plugin-import
+License: MIT  
+https://github.com/benmosher/eslint-plugin-import/blob/master/LICENSE
+
+## Images and components used to run Shopsys Framework on Docker
+### Postgres
+License: PostgreSQL License  
+https://www.postgresql.org/about/licence/
+
+### Nginx
+License: BSD-2-Clause  
+http://nginx.org/LICENSE
+
+### Redis
+License: BSD-3-Clause  
+https://redis.io/topics/license
+
+### Erikdubbelboer/phpredisadmin
```

Comment:
Sometimes, there is the name of the project, and sometime the image name (phpRedisAdmin / Erikdubbelboer/phpredisadmin) - can we make it more consistent? The list is quite hard to read this way.

Reply:
I worked on unification as far as possible, check it :)